### PR TITLE
CalendarToday: Expand triggering

### LINKED ISSUE #3682: Calendar: Should take short hand queries and target year
I have noticed instant answer pops up only for specific queries.

I tried 
- feb calendar - No IA
- February calendar 2017 - No IA
- february calendar - IA

IA should accommodate all these use cases since it is generally expected for people to use short hands for month names and also expect target year.  

---

IA Page:  http://duck.co/ia/view/calendar_today 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Mailkov

### PR DESCRIPTION
**Pull request by AlterationBrick**
## Type of Change

- [ ] New Instant Answer
  - [ ] Cheat Sheet
- [x] Improvement
  - [x] Bug fix
  - [ ] Enhancement
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.)
## Description of new Instant Answer, or changes

Changes trigger from startend to any, allowing phrases like "february calendar 2017" to trigger
## Related Issues and Discussions

Fixes #3682 
## People to notify

@mintsoft @pavkum

---

Forum Topic:

IA Page: https://duck.co/ia/view/calendar_today

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/CalendarToday.pm [hunk 1]
```diff
@@ -12,7 +12,7 @@ with 'DDG::GoodieRole::Dates';
 zci answer_type => 'calendar';
 zci is_cached   => 0;
 
-triggers startend => 'calendar', 'cal';
+triggers any => 'calendar', 'cal';
```
Comment (sahildua2305) [inline lib/DDG/Goodie/CalendarToday.pm]: Good catch! @AlterationBrick

Can you please add some tests verifying the expected behavior?

Comment (AlterationBrick) [inline lib/DDG/Goodie/CalendarToday.pm]: @sahildua2305 I'm afraid I can't get any sort of modifications to the test file to work without it failing, including the duplication of one of the tests. Someone with an actual understanding of Perl should probably take this on. I'm simply trying to add a check for "november calendar 2009" which is the same month the other tests use.

**Comment by moollaza** [comment]
This PR is being closed due to inactivity. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**